.changeset/gorgeous-comics-notice.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lg-tools/storybook-addon': minor
+'@lg-tools/storybook': minor
```

Comment:
The `/storybook` package is deprecated. We don't need to update it (I just haven't removed it yet)

Reply:
Updated!